code/_core/obj/item/clothing/head/hats/witchunter.dm

```diff
@@ -1,16 +1,16 @@
-/obj/item/clothing/head/hat/witchunter

```

Comment:
I don't know why the witchhunter file was changed here. You might be experiencing an issue with newlines. See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings on how to fix it.

Reply:
i have zero idea how to fix this